src/main/java/net/wiseoldman/WomUtilsPlugin.java

```diff
@@ -1178,8 +1245,8 @@ private String compareChanges(Map<String, GroupMembership> oldMembers, Map<Strin
 
 		int membersRemoved = oldMembers.size() + membersAdded - newMembers.size();
 
-		return String.format("Synced %d clan members. %d added, %d removed, %d ranks changed.",
-			newMembers.size(), membersAdded, membersRemoved, ranksChanged);
+		return String.format("Synced WOM group members. %d added, %d removed, %d ranks changed. (%d ranks ignored).",
+			newMembers.size(), membersAdded, membersRemoved, ranksChanged, ignoredRanks.size());
```

Comment:
This shows the total amount of members synced as being added. I'm guessing `ranksChanged` will be shown in `(%d ranks ignored)` instead of  `ignoredRanks.size()`.
![bild](https://github.com/wise-old-man/wiseoldman-runelite-plugin/assets/5983417/1c4eaf23-684c-4db0-ba6e-60bbf173e48e)

A suggestion would be to have something like this instead if we think showing the amount of members synced is valuable. `Synced %d members: %d added, %d removed, %d ranks changed. (%d ranks ignored).`

Was there a particular reason for not stating how many players were synced anymore?

Reply:
No reason besides it's the text that was provided it was meant to become. 

If it should be something else I'm happy to change it to what ever.

Reply:
A little discussion around this on Discord and we think this would make sense to have.
`Synced %d clan members. %d added, %d removed, %d ranks changed, %d ranks ignored.`